types/react-navigation/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for react-navigation 1.5
+// Type definitions for react-navigation 2.0.1
```

Comment:
Version numbers in the header should only include major.minor.

Reply:
When upgrading the major version, please follow the steps outlined here: https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version

Reply:
I made a new push following your documentation, I hope it will be fine :)